workers/unity/Assets/Gdk/Core/MutableView/MutableView.cs

```diff
@@ -17,7 +17,6 @@ public class MutableView
         public Dictionary<int, ComponentTranslation> TranslationUnits;
         public Action<Entity, long> AddAllCommandRequestSenders;
```

Comment:
make private. Only used in the mutable view

Reply:
We should also be making all of these readonly and initialising them at declaration time unless there is a reason we need to new up another one later.